components/blitz/src/omero/gateway/Gateway.java

```diff
@@ -142,7 +142,7 @@
     public static final String PROP_STATELESS_SERVICE_CREATED = "PROP_STATELESS_SERVICE_CREATED";
 
     /** Regex matching ICE Session ID */
-    private static final String ICE_SESSION_ID_REGEX = "[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}";
+    public static final String ICE_SESSION_ID_REGEX = "[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}";
```

Comment:
Interesting. Might be worth moving this to an Ice constant and using this server-side as well.

Reply:
The documentation says that each Ice object has an UUID and so does the session ID look like, although I didn't find anything in the documentation which explicitely says that the session ID is an UUID, too.

Reply:
Hm, actually I could also just use Java's builtin `UUID.fromString()` method.

Reply:
> although I didn't find anything in the documentation which explicitely says that the session ID is an UUID, too.

Exactly. It's not *yet* part of the contract.